src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicyConfig.ts

```diff
@@ -26,10 +26,11 @@ export default class VideoPriorityBasedPolicyConfig {
   private referenceBitrate: number = 0;
 
   /**
-   * Additional server side features to be enabled for network adaption. This
-   * may be overridden by the server. The default may be changed in future releases.
+   * Additional server side features to be enabled for network adaption. The only currently
+   * recommended value is `ServerSideNetworkAdaption.BandwidthProbingAndRemoteVideoQualityAdaption`
    */
-  serverSideNetworkAdaption = ServerSideNetworkAdaption.Default;
+  serverSideNetworkAdaption =
```

Comment:
Just checking if this is a backward compatible change from builder APIs + behavior perspective and whether we need something to revert this back at runtime if builders face issues?

Reply:
TBH I don't think many customers use priority policy without SSNA. Unfortunately since it is client side we don't actually know the number, but given that we don't get any complaints about it (and there is plenty to complain about) I think the number isn't high (and they probably aren't updating the SDK quickly anyways). Customers can of course easily switch to None if they are happy with what they got, but TBH I want to eventually remove the ability to set None (since it isn't technically a change in behavior, I don't see much issue with this).

That being said, I do technically have that knob on the backend, but it's not really a button i want to press given the existing policy has a lot of issues.

Reply:
Cool I am fine with this approach, probably we can leverage client events to help us here in future.